components/common/ButtonEmitWithDropdown.vue

```diff
@@ -4,7 +4,12 @@
       tabindex="0"
       :class="{ open: open }"
       class="flex items-center small-text relative w-full py-1 pl-3 pr-3"
-      @click="toggleOpen"
+      @click="
+        (e) => {
```

Comment:
nitpick(non-blocking): always use full descriptive names in the future, this could also be an error (e is event commonly used for errors)